packages/valory/skills/task_execution/skill.yaml

```diff
@@ -69,11 +69,45 @@ models:
   params:
     args:
       agent_index: 0
-      api_keys: {}
-      tools_to_package_hash: {}
+      api_keys:
+        openai:
+        - dummy_api_key
+        stabilityai:
+        - dummy_api_key
+        google_api_key:
+        - dummy_api_key
+        google_engine_id:
+        - dummy_api_key
+      tools_to_package_hash:
```

Comment:
left over from testing?

Reply:
No, kept them so it does not have to be empty. Same with other configs